### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by tkone2018** [Fafa-DL/Awesome-Backbones#78]
你好，可以帮忙看个问题吗
![image](https://github.com/Fafa-DL/Awesome-Backbones/assets/41476675/4cbae8de-f2ca-48b5-84b1-7af227097033)
 val_loss是越来越大的
data_cfg = dict(
    batch_size = 64,
    num_workers = 4,
    train = dict(
        pretrained_flag = True,
        pretrained_weights = '/home/tangkai02/person_race/Awesome-Backbones-main/weights/efficientnetv2-b2.pth',
        freeze_flag = False,
        freeze_layers = ('backbone',),
        epoches = 50,
    ),
    test=dict(
        ckpt = '',
        metrics = ['accuracy', 'precision', 'recall', 'f1_score', 'confusion'],
        metric_options = dict(
            topk = (1,5),
            thrs = None,
            average_mode='none'
    )
    )
)

# batch 32
# lr = 0.1 *32 /256
# optimizer
optimizer_cfg = dict(
    type='SGD',
    lr=0.01 * 32/256,
    momentum=0.9,
    weight_decay=1e-4)

# learning 
lr_config = dict(type='StepLrUpdater', step=2, gamma=0.973, by_epoch=True)
